pkg/state/impl/store/encryption/marshaler_test.go

```diff
@@ -0,0 +1,166 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+package encryption_test
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/cosi-project/runtime/pkg/resource"
+	"github.com/cosi-project/runtime/pkg/resource/protobuf"
+	"github.com/cosi-project/runtime/pkg/state/conformance"
+	"github.com/cosi-project/runtime/pkg/state/impl/store"
+	"github.com/cosi-project/runtime/pkg/state/impl/store/encryption"
+)
+
+func TestMain(m *testing.M) {
+	err := protobuf.RegisterResource(conformance.PathResourceType, &conformance.PathResource{})
```

Comment:
can't we do this via a more regular `init` function?

Reply:
Hmm, there can be several `init`s, while there can be only one `TestMain` per package. But I agree - its overcomplication in this case.